static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -46,6 +50,7 @@ function InviteMembersModal({
     initialData,
     organization,
     source,
+    setError,
```

Comment:
Let's move the `useState` inside of the `useInviteModal` and have it return the `error`